tests/test-marrainage.js

```diff
@@ -362,6 +363,58 @@ describe('Marrainage', () => {
         /* eslint-disable global-require */
         const { reloadMarrainageJob } = require('../schedulers/marrainageScheduler');
         this.clock.tick(1001);
+        this.listener = (response, obj, builder) => {
+          if (obj.method !== 'update') {
+            return;
+          }
+          knex('marrainage').select().where({ username: staleRequest.username })
+          .then((res) => {
+            res[0].count.should.equal(2);
+          })
+          .then(() => knex('marrainage').select().where({ username: validRequest.username }))
+          .then((res) => {
+            res[0].count.should.equal(1);
+            reloadMarrainageJob.stop() 
+            knex.off('query-response', this.listener) // remove listener else it runs in the next tests
+          })
+          .then(done)
+          .catch(done);
+        }
+        knex.on('query-response', this.listener)
+      });
+    });
+
+    it('should reload stale marrainage requests of edge case exactly two days ago at 00:00', (done) => {
+      var dateStaleRequest = new Date(new Date().setDate(new Date().getDate() - 2))
```

Comment:
```suggestion
      let dateStaleRequest = new Date(new Date().setDate(new Date().getDate() - 2));
```
Pour que ça passe pour le ESLint on peut mettre `let` et point-virgule